build.sbt

```diff
@@ -522,7 +531,8 @@ lazy val `proxy-tests` = (project in file("proxy/proxy-tests"))
   .settings(
     common,
     name := "cloudstate-proxy-tests",
-    fork in Test := true,
+    fork in Test := System.getProperty("RUN_STRESS_TESTS", "false") == "true",
```

Comment:
@jroper I thought this sensible, since running stress tests on every commit/PR seems wasteful right now